docs/rules.md

```diff
@@ -72,51 +72,16 @@ You can also configure the collection methods which this rule
 checks for. By default, all collection methods are checked.
 To only enable a specific set of methods, you could set the
 `noUnnecessaryCollectionCallOnly` configuration key. For example:
-```neon
+```
 parameters:
     noUnnecessaryCollectionCallOnly: ['count', 'first']
 ```
 will only throw errors on the `count` and `first` methods.
 The inverse is also configurable, to not throw an exception
 on the `contains` method, one could set the following value:
-```neon
-parameters:
-    noUnnecessaryCollectionCallExcept: ['contains']
 ```
-
-## NoUnnecessaryEnumerableToArrayCalls
```

Comment:
Sorry, I didn't mean to confuse you, but the only changes to this file should be the new sections you added in this PR